docker/docker-compose-with-hdfs.yml

```diff
@@ -0,0 +1,305 @@
+---
+version: "3"
+services:
+  compactor-0:
+    image: "ghcr.io/risingwavelabs/risingwave:RW_0801_HDFS_273-x86_64"
```

Comment:
Use a more formal version later?

Reply:
In the dockerfile for specific customers, `<the image you packed>` is used instead.